src/scip.rs

```diff
@@ -10,12 +10,12 @@ use std::collections::BTreeMap;
 use std::ffi::{c_int, CStr, CString};
 use std::mem::MaybeUninit;
 use std::rc::Rc;
+use scip_sys::SCIP_SOL;
 
 #[non_exhaustive]
 #[derive(Debug)]
 pub(crate) struct ScipPtr {
     pub(crate) raw: *mut ffi::SCIP,
```

Comment:
I think it might be good for the future to have this eventually a NotNull. But it is outside this pull's scope

Reply:
Thanks, I added issue #156 to track this.